Homework-Feb-29/BWT/SuffixArray.cs

```diff
@@ -0,0 +1,44 @@
+namespace BWT;
+
+static class SuffixArray
+{
+    public static List<int> BuildSuffixArray(string sourceString)
+    {
+        var suffixArray = new List<int>();
+        for (var i = 0; i < sourceString.Length; ++i)
+        {
+            suffixArray.Add(i);
+        }
+        SortSuffixArray(sourceString, suffixArray);
+        return suffixArray;
```

Comment:
Обычно все таки принято ставить пустую строку перед `return`

```suggestion
        SortSuffixArray(sourceString, suffixArray);
        
        return suffixArray;
```